### PR TITLE
fix: overflow menu should be closed when click it once again

### PR DESCRIPTION
**Pull request by trainto**
Normally when hamburger menu button clicked while overflow menu is on, users expect it to be closed.

### CODE REVIEW

File: web/src/transmission.js [hunk 1]
```diff
@@ -141,7 +141,9 @@ export class Transmission extends EventTarget {
           this.setCurrentPopup(new MoveDialog(this, this.remote));
           break;
         case 'show-overflow-menu':
-          {
+          if (this.popup instanceof OverflowMenu) {
+            this.popup.close();
```
Comment (ckerr) [inline web/src/transmission.js:145]: _(nit)_ I think popup.close() would work here due to the popupCloseListener but for consistency with the rest of the places that replace the popup, `setCurrentPopup(null)` would be slightly better here

```suggestion
            this.setCurrentPopup(null);
```

**Review by ckerr — COMMENTED** [review]
Thanks for finding this bug and for tracking it down! It's great to see that people are using this new code already :smile_cat:

File: web/src/transmission.js (hunk 1 above)
Comment (trainto) [reply web/src/transmission.js]: @ckerr commit replaced as you suggested :)

**Comment by trainto** [comment]
@ckerr Thank you for your time and review. I replaced the commit as you suggested.

**Review by ckerr — APPROVED** [review]

**Pull request merged**